src/components/Card/index.js

```diff
@@ -3,9 +3,21 @@ import './Card.css';
 import Button from '../Button'
 
 export default class Card extends Component {
+    static propTypes = {
+        text: PropTypes.string.isRequired,
+        id: PropTypes.int.isRequired,
```

Comment:
This is going to error because PropTypes isn't defined. you need
`import PropTypes from 'prop-types'`